Python/Product/TestAdapter/TestContainerDiscovererProject.cs

```diff
@@ -241,7 +241,7 @@ class TestContainerDiscovererProject : ITestContainerDiscoverer, IDisposable {
             var projInfo = new ProjectInfo(pyProj);
             _projectMap[projInfo.ProjectHome] = projInfo;
             var files = FilteredTestOrSettingsFiles(vsProject);
-            UpdateSolutionTestContainersAndFileWatchers(files, projInfo, isAdd: true);
+            UpdateContainersAndListeners(files, projInfo, isAdd: true);
```

Comment:
rename.
previous name was from when openfolder and solution logic were both in this file